content/docs/api_reference/config/v1alpha1.md

```diff
@@ -1,29 +1,23 @@
----
-title: "Config v1alpha1 API Reference"
-description: "Config v1alpha1 API reference documentation."
-type: docs
----
-
```

Comment:
We should keep the header.

Also please generate the docs for config/v1alpha2 and policy/v1alpha1.